core/src/main/java/jenkins/triggers/ReverseBuildTrigger.java

```diff
@@ -137,7 +137,8 @@ private boolean shouldTrigger(Run upstreamBuild, TaskListener listener) {
         return result != null && result.isBetterOrEqualTo(threshold);
     }
 
-    @Override public void buildDependencyGraph(final AbstractProject downstream, DependencyGraph graph) {
+    @Override
+    public void buildDependencyGraph(final AbstractProject downstream, DependencyGraph graph) {
```

Comment:
Gratuitous whitespace-only changes should be avoided to minimize work with cherry-picking etc.